.github/workflows/ci_linting.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Linters
+
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  linters:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        command:
+          - name: cppcheck
+            run: |
+            source ./codebuild/bin/s2n_setup_env.sh
+            ./codebuild/bin/install_cppcheck.sh "$CPPCHECK_INSTALL_DIR"
+            ./codebuild/bin/run_cppcheck.sh "$CPPCHECK_INSTALL_DIR"
+          - name: copyright_mistakes
+            run: |
+            source ./codebuild/bin/s2n_setup_env.sh
+            ./codebuild/bin/copyright_mistake_scanner.sh
+          - name: grep_simple_mistakes
+            run: |
+            source ./codebuild/bin/s2n_setup_env.sh
+            ./codebuild/bin/grep_simple_mistakes.sh
+          - name: style checks
+            run: |
+            source ./codebuild/bin/s2n_setup_env.sh
+            ./codebuild/bin/run_kwstyle.sh
+            ./codebuild/bin/cpp_style_comment_linter.sh
+    steps:
+      - uses: actions/checkout@v2
+      - name: ${{ matrix.command.name }}
+        run: ${{ matrix.command.run }}
```

Comment:
It did not occur to me to look to see if you could do that. Nice!

Reply:
You can't :-|, removed

Reply:
:(